scripts/deploy-chart-test.sh

```diff
@@ -61,7 +64,7 @@ up() {
   $VECTOR_TEST_HELM repo add "$CUSTOM_HELM_REPO_LOCAL_NAME" "$CHART_REPO" --force-update || true
   $VECTOR_TEST_HELM repo update
 
-  $VECTOR_TEST_KUBECTL create namespace "$NAMESPACE"
+  $VECTOR_TEST_KUBECTL create namespace "$NAMESPACE" --dry-run -o yaml | $VECTOR_TEST_KUBECTL apply -f -
```

Comment:
This allows us to try and create the same namespace multiple times, which happens in the `vector.rs` tests.